### PR TITLE
configure the VS Code Debugger for the development

### PR DESCRIPTION
**Pull request by toolowner**
Suggestion on using VS Code - Debugger for Chrome for the development

### CODE REVIEW

**Comment by thijstriemstra** [comment]
What about adding this json file to the repostitory @toolowner?

**Comment by toolowner** [comment]
Good idea :+1:  I will add it.

I wanted to do that but I was afraid that some people don't prefer that way.

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```
Comment (thijstriemstra) [inline .vscode/launch.json]: it's weird there are comments in this json file (rendering it invalid), did you add those manually or?

Comment (toolowner) [reply .vscode/launch.json]: That comment was generated by VS Code.

Comment (toolowner) [reply .vscode/launch.json]: I will delete it.

**Pull request merged**